torch/testing/_internal/common_methods_invocations.py

```diff
@@ -485,7 +486,7 @@ def __init__(self,
                  sample_inputs_func=None,  # function to generate sample inputs
 
                  # the following metadata relates to dtype support and is tested for correctness in test_ops.py
-                 dtypes=floating_types(),  # dtypes this function is expected to work with
+                 dtypes=None,  # dtypes this function is expected to work with
```

Comment:
Let's update this comment to be "dtypes this function is expected to work with on the CPU"

Reply:
Will update it but a comment : This is still generic for all devices (if their respective fields are not set right), so I think the existing `dtypes this function is expected to work makes ` sense. 🤔

Reply:
That's a good point; what about merging our ideas with something like "dtypes this function works with on the CPU, inherited by other device types that don't specify their own dtypes"?

Reply:
That sounds good. Thanks!